app/app/api/api_v1/endpoints/users.py

```diff
@@ -170,21 +170,28 @@ def update_user_profile_picture(
     db: Session = Depends(deps.get_db),
     user_id: int,
     image: UploadFile = File(...),
-    current_admin: models.Admin = Depends(deps.get_current_active_admin_with_permission("user")),
```

Comment:
Umm I see this leading to students uploading NSFW / rendum wallpapers as PP.

Reply:
Hmm tho idk who dumb enough to put nsfw image for their own acc knowing that only they can do that and they can't gib excuses

Reply:
Final year students incase account not disabled maibi

Reply:
But hab same issue with all updates student can do to their own data.
Basically inside the `current_user.id == user_id` wala IF we need to send a request to a valid admin, whose "allow" button in the dialog(maybe) just updates the pp of requesting user with their given pp

Reply:
Yeah that's the issue, we wanted to not deal with request/review wala stuff.

Reply:
I mean, just sending mail to a valid admin with details, requested pp, and a link to another endpoint which just changes pp of the requestor

Reply:
More DB stuff tho, need a table to store these temporarily requested ones, and then copy to regular.
Idm whatever though, @ksdfg what do you want

Reply:
Tempo ke liye db kyu. imgur links

Reply:
1. We not using imgur
2. That imgur link would also need to be saved na, admin not gonna download from mail and manually edit user ka xD

Reply:
Not a priority